documents/description.md

```diff
@@ -1,9 +1,19 @@
 # Project description
 
 ## Motivation
+Many modern languages are not well designed for parallel computing (except ones like Chapel). You often have to separete parallel code from serial code via a for loop, a function call, or some other means. To further complicate things, you often need to call special functions, use strange syntax, and/or switch to certain compilers just to get off the ground. My goal is to create an easy-to-use library that allows you to make it easy to include implicitly parallel code right next to your standard serial code so that you can easily get parallel scpeedups without a lot of the mess that comes with traditional parallel languages or libraries. To do this, I plan on making a DSL that allows you to work with vectors like you would normal C++ variables. This will allow programmers to implicitly do operations on many variables with code that will look extraordinarily familiar.
 
 ## Language domain
+This DSL will target vector-level parallelism. While it won't be best for every kind of parallel problem, it should work well for problems that involve doing largely the same operations to large sets of data and potentially reducing the results down to a single number (like doing dot products on vectors containing hundreds or thousands of variables).
+
```

Comment:
A bit more detail would be good, especially for the uninitiated: what is "vector-level parallelism"?